### ISSUE TITLE
iNDS Multiplayer

### ISSUE THREAD

**Issue by MaquinaDelMisterio** [WilliamLCobb/iNDS#150]
I have used iNDS on my iPad Pro and it works flawlesly, at 60fps. I think it would be great if we could have multiplayer on the emulator.

**Comment by pmp174** [comment]
I am not 100% sure but I believe the creators DeSmuMe said that the core would never support it. ( they kind of quit trying to implement this after a while.

**Comment by pmp174** [comment]
![image](https://cloud.githubusercontent.com/assets/15956420/15867663/484488b0-2cb2-11e6-97aa-89e0b7ad27b9.png)
Forgive me if I'm wrong but look at the red print. I believe multiplayer requires the wifi emulation.

**Comment by MaquinaDelMisterio** [comment]
You are right. So, i will stop asking for multiplayer. Thanks for the info.

**Comment by essiw** [comment]
This does not mean that it cannot work in some other way, e.g. only for iNDS users. This would however mean lots of programming (and I am not sure if it is feasible, or even worthwhile as I don't know the amount of players per game of iNDS)

**Comment by MaquinaDelMisterio** [comment]
I mean, like local mjltiplayer function. And yes, like you say its a lot of work to be done there to make it work.

**Comment by DarknesGaming** [comment]
The DeSmuME team did get Wi-Fi and Multiplayer functions working, but it was removed because of Nintendo.

**Comment by pmp174** [comment]
So that could hurt the dev.

**Comment by scj643** [comment]
Still think this would be nice and Nintendo shouldn't complain since they were the ones who shut down the Nintendo WFC for every DS title.
